content/enterprise/install/automated/active-active.mdx

```diff
@@ -230,6 +228,43 @@ Finally, confirm the application is functioning as expected when running multipl
 
 Confirm the general functionality of the Terraform Enterprise UI to validate the tokens you added in Step 2 are set correctly. Browse the `Run` interface and your organization's private registry to confirm your application functions as expected.
 
+### Scaling Beyond Two Nodes
+
+Terraform Enterprise supports scaling up to three nodes as part of the Active/Active deployment. When scaling beyond two nodes, you should also carefully evaluate and scale external services, particularly the database server. Scaling to three nodes does not alter the [upgrade requirement](https://www.terraform.io/enterprise/admin/infrastructure/active-active#upgrading-tfe-or-patching-tfe-node-instances) that you scale down to a single node or rebuild entirely.
```

Comment:
Here we spell out "three" and "two" whereas earlier in the document we use digits. This is super nit-picky, but I'm curious if we have a style guide rule to turn to here?

Reply:
such a good point!! I pulled this from our internal style guide: 

"For more formal writing, write out “one” through “nine” and write 10 and above as numerals.
For more technical writing in sentences with technical terms, write most numbers as numerals for ease of reading. Spell out “one” if it’s not used in tandem with another number (like in a range 1 to 4 for example) in the sentence."

As this is technical content, I believe we should write them all as numerals. Let me know if you disagree!

Reply:
I have no objection. Done